pages/_document.js

```diff
@@ -1,41 +1,38 @@
 /* eslint-disable react/jsx-filename-extension */
-import React from 'react';
-import Document, { Html, Main, NextScript } from 'next/document';
-import { ServerStyleSheets } from '@material-ui/core/styles';
+import React from "react";
+import Document, { Html, Head, Main, NextScript } from "next/document";
+import { ServerStyleSheets } from "@material-ui/core/styles";
 
 export default class MyDocument extends Document {
```

Comment:
can you explain the usage of this file ?

Reply:
We followed these steps to setup Material-UI with NEXT initially. 
https://itnext.io/next-js-with-material-ui-7a7f6485f671

The purpose was to remove server side injected css.